sdk/aqueduct/api_client.py

```diff
@@ -325,3 +329,13 @@ def get_node_positions(
         resp_json = resp.json()
 
         return resp_json["operator_positions"], resp_json["artifact_positions"]
+
+    def export_serialized_function(self, work_flow_response: GetWorkflowResponse) -> None:
+        headers = utils.generate_auth_headers(self.api_key)
+        for work_flow_dag in list(work_flow_response.workflow_dags.values()):
+            for operator in list(work_flow_dag.operators.values()):
+                operator_url = self._construct_full_url(
+                    self.EXPORT_FUNCTION_ROUTE % str(operator.id), self.use_https
+                )
+                operator_resp = requests.get(operator_url, headers=headers)
+                work_flow_dag.update_operator_spec(operator, operator_resp)
```

Comment:
ok this is better, but I was really saying this APIClient layer should be as thin as possible - it should literally just take in a single operator id and return the serialized function for that operator. The double-for loop belongs in the caller.

Question: does an operator always only correspond to a single workflow dag? Is there a case where an operator corresponds to multiple workflow dags on the backend. cc @cw75